tutorials/howto/howto_new_profile_version.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: tutorial
+title: How to create and publish a new draft profile
```

Comment:
The tutorial should be added to the Overview page.

Reply:
done